homework/index.js

```diff
@@ -43,8 +65,20 @@
         });
         return;
       }
-      const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      createAndAppend('h1', root, {
+        text: 'HYF Repository',
+        class: 'title',
+      });
+      repos.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      });
+
+      repos.forEach(repo => {
```

Comment:
Optionally, you can combine the sorting and `.forEach()` in a chain:

```js
repos
  .sort((a, b) => a.name.localeCompare(b.name))
  .forEach(repo => renderRepoDetails(repo, repoContainer));
```

Reply:
Thank you for making it shorter. I implemented it.